contrib/systemd/lxa-iobus.service

```
@@ -1,15 +1,18 @@
 [Unit]
 Description=LXA iobus Server
+After=network.target
 
 [Service]
 Type=simple
 ExecStartPre=/usr/bin/mkdir -p /var/cache/lxa-iobus
-ExecStart=/usr/bin/lxa-iobus-server -l WARN --firmware-directory /var/cache/lxa-iobus --host "::" can0_iobus
+ExecStartPre=/usr/bin/ip l set can0_iobus down
+ExecStartPre=/usr/bin/ip l set can0_iobus up
+ExecStart=/usr/venvs/lxa-iobus/bin/lxa-iobus-server -l WARN --lss-address-cache-file /var/cache/lxa-iobus/lss-cache --host "*" can0_iobus
```

```suggestion
ExecStartPre=/usr/bin/ip llink set can0_iobus down
ExecStartPre=/usr/bin/ip link set can0_iobus up
ExecStart=/usr/venvs/lxa-iobus/bin/lxa-iobus-server -l WARN --lss-address-cache-file /var/cache/lxa-iobus/lss-cache --host "*" can0_iobus
```
How is the ``/var/cache/lxa-iobus`` directory created?
If we toggle the can0_iobus interface here every time, why is a on-shot service needed? Timing can also be set at the same time as bringing the interface up.

---

The `/var/cache/lxa-iobus` directory is created here:
https://github.com/linux-automation/lxa-iobus/pull/24/files#diff-d844640adc719b80e871c13445167a4ec798e89c1768f52e00b841ff0c301509R7

The link is toggled here to reset the bus from an eventual `ERROR-PASSIVE` state. I guess this is more a hack. It would probably be better to do this from inside the server.

---

So if we reset the bus in this unit anyway, why not handle timing configuration here as well and avoid the one-shot service?

---

Implemented it as @jluebbe  suggested. Please take another look.